designs/credentialManagement/ssoSupport.md

```diff
@@ -0,0 +1,32 @@
+# AWS SSO Support
+
+## Abstract
+
+This document describes the process of supporting Amazon SSO defined through a shared credentials profile.
+
+## Specification
+
+The [AWS SSO-OIDC service][SsoOidc] is SSO's implementation of the Device Authorization Grant flow as defined in [RFC8628][RFC8628].
+
+## SSO Login Sequence
+![][SsoLoginFlow]
+
+## Cache
+The SSO login flow contains two long lived tokens (`client registration id` and `access token`). These tokens MUST be cached in order to prevent the user from 
```

Comment:
These are "long lived" but 12 hours at the most so they are not really long lived?

Reply:
Client registration is like 90 days?

Reply:
Ah I was looking at the docs for the access token which is much shorter lived